env-defaults

```diff
@@ -143,10 +143,10 @@ DATABASE_URL=
 UPLOAD_BUCKET=local-upload-bucket
 
 # SQS queue used to process work item updates
-WORK_ITEM_UPDATE_QUEUE_URL=http://localstack:4566/queue/work-item-update-queue
+WORK_ITEM_UPDATE_QUEUE_URL=http://localhost:4566/queue/work-item-update-queue
```

Comment:
I think this accidentally reverted a change James made to fix harmony in a box.

Reply:
Yes, it did.